src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
```

Comment:
Avoid using static fields for mutable objects like `Random`. Instead, create an instance of `Random` where it's needed or pass it as a parameter if necessary.